utils/tests_fetcher.py

```diff
@@ -1148,6 +1148,7 @@ def parse_commit_message(commit_message: str) -> Dict[str, bool]:
     "pipelines_torch": r"tests/models/.*/test_modeling_(?!(?:flax_|tf_)).*",
     "tests_hub": r"tests/.*",
     "tests_onnx": r"tests/models/.*/test_modeling_(?:tf_|(?!flax)).*",
+    "tests_non_model": r"tests/[^/]*?/test_.*\.py",
```

Comment:
we should make sure we filter to not pass in the `test_modeling_llama` only `test_modeling_utils` wdyt?

Reply:
This re `r"tests/[^/]*?/test_.*\.py"` won't include `test_modeling_llama` (which is inside `tests/models`), because it exclude the intermediate `/` using `[^/]`

Reply:
ahh okay all good for me then!